spec/factories/notes.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :note do
+    title { Faker::Book.title }
+    content { Faker::Lorem.paragraph }
+    note_type { Note.note_types.keys.sample }
+    user { association :user }
```

Comment:
Esta forma es equivalente también :relaxed: 
```suggestion
    user
```
Esto lo que hace es que utilize el propio Factory de User, y en caso que desees que ese `user` tenga atributos específicos, podes pasárselo como parámetro cuando generes la `note`

Reply:
Ahi cambie para que el Factory de note use el Factory de user.